javascript-loops-practice/javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -3,7 +3,8 @@ function addSuffixToAll(words, suffix) {
   var suffixArray = [];
 
   for (var i = 0; i < words.length; i++) {
-    suffixArray.push(words[i] += suffix);
+    var word = words[i] += suffix;
```

Comment:
This is still affecting the original value. The problem here is that you are incorporating assignment into the concatenation you want to do, which is the primary goal here. If you remove the assignment component and just stick to concatenation, this should work 👍